javascript/javascript2/week1/problemSolvingCardio/findNCountDanishLetter.js

```diff
@@ -0,0 +1,44 @@
+const danishString = "Jeg har en blå bil"; 
+
+const danishString2 = "Blå grød med røde bær"; 
+
+const danishString3 = "19-årige Jakob fra København skal spille VM for USA";
+
+function individualChar(danishString) {
+    let count = 0;
+    let countå = 0;
+    let countø = 0;
+    let countæ = 0;
+    let output = {
+        total: count,
+        å: countå,
+        ø: countø,
+        æ: countæ,
+      }; 
+
+      for (let i = 0; i < danishString.length; i++) {
+        if (danishString[i] === "å") {
+          count++;
+          countå++;
+          output.å = countå;
```

Comment:
This is a correct, and clean solution. I can only recommend you something, since I can tell that you have a good understanding of JS: 
- are the `count` variables necessary? Essentially, you could just increase the value of the `output.countX` variables directly in the `if`s. At the end of the function, before returning you can just calculate the total as the sum of the other three variables in the `output` object. The less code you write, the less code can confuse you later. 😉